src/main/java/com/wavefront/sdk/proxy/WavefrontProxyClient.java

```diff
@@ -181,14 +181,11 @@ private WavefrontProxyClient(Builder builder) {
     scheduler.scheduleAtFixedRate(this, 1, builder.flushIntervalSeconds, TimeUnit.SECONDS);
   }
 
-  @Override
-  public void sendMetric(String name, double value, @Nullable Long timestamp,
-                         @Nullable String source, @Nullable Map<String, String> tags)
-      throws IOException {
+  private boolean setupSendMetric() throws IOException {
```

Comment:
rename this private method to
boolean setupConnection

Reply:
Done